### ISSUE TITLE
PostMan's GUI  is not accessible for blind developers that are using screen reader to work with computer.

### ISSUE THREAD

**Issue by hamidrezaabroshan** [a85/POSTMan-Chrome-Extension#9569]
Hi,
I'm a blind java developer and wanted to test some rest api and find
out that the Ui of this app is not accessible for me.
I'm using nvda an open source project to work with computer and you
can download a coppy from [nvda project](https://www.nvaccess.org/)
This app reads every thing that appears on the screen and I can tell
it that which part of Screen I want to be read by using shortcut.
I'll be glad to help you to make the UI accessible.
Thanks.[](url)

**Comment by pferrucci** [comment]
+1

This is a shame blind people are not supported.
Also, I don't understand why people code like this in this 21 century. I spent several hours to write an AutoIt program to install Postman completely automatically. It was a long process because of the non-standard GUI.
Postman is an OpenSource program with a closed-minded GUI. :-(

**Comment by balajip8811** [comment]
Please support us . please make it accessible to screen readers. it does not take a lot of time.  we  can do it in a sprint.  we have done a lot of projects for eu and canada and usa projects  where  accessibility is required from first day.

**Comment by sslotay** [comment]
I agree. I am in college doing a network course in my cs program. I have a tough time using postman. It's not like accessiblity needed special work especially for web, this is just terrible, lazy, bad code and semantic.
Never in my lave would I have thought that Cisco Packet Tracer, made in QT which is known for its accessibility issues, would really be a truly enjoyable experience compared to this ui. And no, web is not the problem at all. Web is probably the simpler to just restructurate enough so accessibility just happens by design. But especially for an open core software, postman is a disappointment.
